packages/venia-concept/webpack.config.js

```diff
@@ -91,5 +93,28 @@ module.exports = async env => {
         })
     ];
 
-    return config;
+    const serverConfig = Object.assign({}, config, {
+        target: 'node',
+        output: {
+            ...config.output,
+            filename: '[name].[hash].SERVER.js',
+            strictModuleExceptionHandling: true
+        },
+        devtool: false,
+        optimization: {
+            minimize: false
+        },
+        plugins: [...config.plugins]
+    });
+
+    // remove HtmlWebpackPlugin
+    serverConfig.plugins.pop();
```

Comment:
Is this position guaranteed? Seems a little fragile, but comfortable with this if we control both sides.

Reply:
No; it's not guaranteed, and it is fragile. I've updated the code to filter by plugin name rather than index.